user-org-service/app/controllers/logsmanager/validator/LogValidator.java

```diff
@@ -0,0 +1,44 @@
+package controllers.logsmanager.validator;
+
+import org.apache.commons.lang3.EnumUtils;
+import org.sunbird.util.LoggerEnum;
+import org.sunbird.util.ProjectLogger;
+import org.sunbird.util.UserOrgJsonKey;
+import org.sunbird.util.request.Request;
+
+public class LogValidator {
+
+  /**
+   * This method will validate the log levels supported by service or not
+   *
+   * @param level
+   * @return boolean
+   */
+  protected static boolean validateLogLevels(String level) {
+    ProjectLogger.log(
+        String.format(
+            "%s:%s: level to be set %s",
+            LogValidator.class.getSimpleName(), " validateLogLevels", level));
+    return EnumUtils.isValidEnum(LoggerEnum.class, level.toUpperCase());
+  }
+
+  /**
+   * This method will validate wheather required requestBody logLevel is present or not
+   *
+   * @param request
+   * @return boolean
+   */
+  public static boolean isLogParamsPresent(Request request) {
+    return request != null && request.get(UserOrgJsonKey.LOG_LEVEL) != null ? true : false;
```

Comment:
use (request != null && request.get(UserOrgJsonKey.LOG_LEVEL)) it will have more visibility to enduser